packages/@expo/fingerprint/src/sourcer/Expo.ts

```diff
@@ -28,11 +28,11 @@ export async function getExpoConfigSourcesAsync(
   let loadedModules: string[] = [];
   const ignoredFile = await createTempIgnoredFileAsync(options);
   try {
-    const { stdout } = await spawnAsync(
-      'node',
-      [getExpoConfigLoaderPath(), path.resolve(projectRoot), ignoredFile],
-      { cwd: __dirname }
-    );
+    const { stdout } = await spawnAsync('node', [
+      getExpoConfigLoaderPath(),
+      path.resolve(projectRoot),
+      ignoredFile,
+    ]);
```

Comment:
```suggestion
    const { stdout } = await spawnAsync(
      'node',
      [getExpoConfigLoaderPath(), path.resolve(projectRoot), ignoredFile],
      { cwd: projectRoot }
    );
```

nit: using `projectRoot` probably may be explicit and clear?

Reply:
makes sense, thx 👍